spec/fixtures/marc_to_solr/local_subject_heading.mrx

```diff
@@ -50,7 +50,6 @@
     </datafield>
     <datafield tag="650" ind1=" " ind2="0">
       <subfield code="a">Illegal aliens</subfield>
-      <subfield code="z">Europe.</subfield>
```

Comment:
why is 'z' removed from the fixture?

Reply:
I removed it, because after Change The Subject runs, this field will be identical to the local heading field.  I wanted them to be slightly different so we could confirm that both are indexed.